### DISCUSSION TITLE
Texture Map copy from one face to other?

### DISCUSSION THREAD

**Discussion by serious-coder20** [Garux/netradiant-custom#125] [Q&A]
Hi, I am just trying to figure out how to copy textures (along with projection mapping) from one brush face to another in netradiant?

**Comment by Garux** [comment]
menu.help.general.mouse shortcuts

**Reply by serious-coder20** [reply]
k thanks. Is it possible to do the following (how) in the [20220211] Release

1: Insert custom vertex in the brush
2: Combine two brushes (without deleting the splitting edge)?

**Comment by Garux** [comment]
1. click in vertex mode
2. merge command (idk what `without deleting the splitting edge` implies)

**Comment by Garux** [comment]
https://garux.github.io/NRC/

**Reply by serious-coder20** [reply]
I saw that link. But would be helpful if u can explain in more detail for this following image :

![26brushCurve](https://user-images.githubusercontent.com/72692966/206906170-a7926ec0-5380-478f-8d89-f1fb514f2e73.gif)

> Which base shape u used to create the cylinder?
> How did extrude multiple selected faces?
> How to join the corners to fill the gap? (in the end)

Thanks.

**Reply by Garux** [reply]
1. brush.prism
2. selected, extruded
3. 'csg wrap merge'

**Reply by serious-coder20** [reply]
I am unable to extrude individual selected brush faces. Can u provide more detail?

**Reply by serious-coder20** [reply]
Hi.. i guess i found out. Its under csg extrusion tool. thanks anyways.

**Reply by serious-coder20** [reply]
I forgot to ask, in one of your videos u used a red selection tool to select and delete some brushes..

[https://youtu.be/qA9eNOkD1iY](url) 
(Time in video: 2:04 to 2:12) 

Can u tell how to do this kind operation?

**Reply by Garux** [reply]
CSG subtract
